src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDateTime by; 
+
+    /**
+     * Deadline task constructor
+     *
+     * @param description description of command
```

Comment:
it's a little hard to read without any separators to separate the parameter name and your comment on what it means. perhaps you can consider @param description: description of command for example.